interop/applier.go

```diff
@@ -28,6 +28,8 @@ var interopPredeploys = []common.Address{
 	predeploys.CrossL2InboxAddr,
 	predeploys.L2toL2CrossDomainMessengerAddr,
 	predeploys.L1BlockAddr,
+	common.HexToAddress("0x4200000000000000000000000000000000000024"),
+	common.HexToAddress("0x4200000000000000000000000000000000000025"),
```

Comment:
will be able to import these from predeploys package once https://github.com/ethereum-optimism/optimism/pull/11927 merges